### PR TITLE
add warning page for securedrop v2 onion address

### PR DESCRIPTION
**Pull request by itsibitzi**
Very quick hidden service which we can host on the old securedrop v2 address once we take it down.

It informs users to check theguardian.com/securedrop for renewed instructions.

Look of the page:
image

### CODE REVIEW

**Comment by itsibitzi** [comment]
The screenshot URL and copy URL are different because the current onion URL is not the securedrop one, because it's still active 😊

File: sd-warning/cloudformation/sd-warning.yaml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Allow HTTP and HTTPS inbound connections
+      VpcId:
+        Ref: VpcId
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 8080
+          ToPort: 8080
+          CidrIp: 127.0.0.1/0
```
Comment (katebee) [inline sd-warning/cloudformation/sd-warning.yaml:62]: I am not sure if we need this ingress rule? localhost traffic should not be affected by a security group

File: sd-warning/cloudformation/sd-warning.yaml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      - PROD
+      - CODE
+      - DEV
+    Default: PROD
+
+Resources:
+  InstanceSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Allow HTTP and HTTPS inbound connections
```
Comment (katebee) [inline sd-warning/cloudformation/sd-warning.yaml:55]:
```suggestion
      GroupDescription: Allow HTTP inbound connections
```

File: sd-warning/static/public.css [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+    border-radius: 100px;
+    text-transform: none;
+    box-shadow: none;
+    margin: 6px 0;
+}
+
+.sd-maintenance {
+    background-color: #cc2b12;
+}
+T
```
Comment (katebee) [inline sd-warning/static/public.css:314]:
```suggestion

```

Comment (katebee) [reply sd-warning/static/public.css:314]: Are you just checking I am paying attention 😛

**Review by katebee — APPROVED** [review]
Couple of minor points but nothing preventing a 👍 from me.
Guessing we are using an AMI with tor baked in?

**Comment by itsibitzi** [comment]
I will update the readme to include a reference to the tor/nginx AMI

**Comment by katebee** [comment]
Shall we???

Screenshot 2021-03-25 at 14 29 15

**Pull request merged**